coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -85,7 +85,41 @@ function createBundleEntries(cancellationResponse: CancellationResponse) {
     fhirResponsiblePartyPractitioner,
     fhirResponsiblePartyOrganization,
     fhirResponsiblePartyPractitionerRole
-  ].map(convertResourceToBundleEntry)
+  ]
+
+  if (cancellationResponse.performer) {
+    const performerAgentPerson = cancellationResponse.performer.AgentPerson
+    const {
+      fhirPractitioner: fhirPerformerPractitioner,
+      fhirPractitionerRole: fhirPerformerPractitionerRole,
+      fhirOrganization: fhirPerformerOrganization
+    } = convertAgentPerson(performerAgentPerson)
+    const performerPractitionerId = fhirPerformerPractitioner.id
+    const performerOrganizationCode = performerAgentPerson.representedOrganization.id._attributes.extension
+    const performerOrganizationName = performerAgentPerson.representedOrganization.name._text
+    fhirMedicationRequest.dispenseRequest = createDispenserInfoReference(
+      performerPractitionerId, performerOrganizationCode, performerOrganizationName
+    )
+    bundleResources.push(fhirPerformerPractitioner, fhirPerformerPractitionerRole, fhirPerformerOrganization)
```

Comment:
As far as I can tell, we only need the Practitioner in the bundle. The others appear to be unreferenced

Reply:
Actually looking at the IG, `MedicationReference.dispenseRequest.performer.extension("https://fhir.nhs.uk/R4/StructureDefinition/Extension-DispensingPerformer").valueReference` should point to the PractitionerRole, not the Practitioner (and therefore the Practitioner and Organization would all be referenced too)